Podfile

```diff
@@ -1,5 +1,5 @@
 source 'https://github.com/CocoaPods/Specs.git'
-platform :ios, '12.0'
+platform :ios, '12.1'
```

Comment:
Any reason to go to 12.1? The project is still at 12.0.

Reply:
Sorry, was wrangling with making the simulator work on M1 and changed this. Not necessary, will revert

Reply:
Feel you 😂 I threw some bad words at the simulator in the past..